Cargo.toml

```diff
@@ -17,7 +17,7 @@ tempfile = "3.3"
 xdg = "2.4"
 
 [dependencies]
-clap = { version = "3.2", features = ["cargo"] }
+clap = { version = "4.0", features = ["cargo"] }
```

Comment:
Tiny nit, but it might be worth considering bumping to the latest minor here (`4.0.9`), since they had a couple of bug fixes since 4.0.

Reply:
I have very mixed feelings about this. But the way I have been leaning lately is to not include patch versions in Cargo.toml files. Tracking patch versions tends to introduce a lot of noise, and I prefer changes to the Cargo.toml file to be "significant," e.g., addition/removal of a dependency.